core/src/com/unciv/models/ruleset/unique/Conditionals.kt

```diff
@@ -99,6 +99,48 @@ object Conditionals {
             return compare(statReserve, lowerLimit * gameSpeedModifier, upperLimit * gameSpeedModifier)
         }
 
+        fun getCountableAmount(countable: String): Float? {
+            if (countable.toIntOrNull() != null) return countable.toFloat()
+            if (countable.toFloatOrNull() != null) return countable.toFloat()
+            if (countable == "year") return gameInfo!!.getYear(gameInfo!!.turns).toFloat()
```

Comment:
A lot of assumptions in this dunction that things aren't null - we should be checking!
gameInfo, civInfo, etc - they can be null, what happens then?